requirements.in

```diff
@@ -3,6 +3,6 @@ pandas == 1.0.*
 gunicorn
 requests
 lxml
-bs4
+beautifulsoup4
```

Comment:
Ah good catch!  (it was working because the author added a `bs4` package on PyPI that depends on `beautifulsoup4`)